misc/assets/mail.svg

```diff
@@ -0,0 +1,60 @@
+<svg version="1.1" viewBox="0 0 1000 400" xmlns="http://www.w3.org/2000/svg">
+  <style>
+      text {
+        font-family: sans-serif;
+      }
+      .caption {
+        font-size: 12pt;
+        text-anchor: middle;
+        dominant-baseline: central;
+      }
+      .note {
+        font-size: 9pt;
+        text-anchor: middle;
+        dominant-baseline: ideographic;
+      }
+      .relation {
+        fill: none;
+        stroke: black;
+        stroke-width: 1pt;
+      }
+      .relation.from-many {
+        marker-start: url(#relation-many)
+      }
+      .relation.polymorphic {
+        marker-start: url(#relation-polymorphic)
+      }
+  </style>
+
+  <defs>
+    <marker id="relation-many" orient="auto" markerWidth="6" markerHeight="6" refX="0" refY="3">
+      <circle cx="3" cy="3" r="3" />
+    </marker>
+    <marker id="relation-polymorphic" orient="auto" markerWidth="6" markerHeight="6" refX="0" refY="3">
+      <circle cx="3" cy="3" r="3" fill="white" stroke="black" />
+    </marker>
+    <g id="model">
+      <rect class="box" x="-100" y="-20" width="200" height="40" fill="none" stroke="black" stroke-width="1pt" />
+    </g>
+    <g id="external">
+      <rect class="box" x="-100" y="-20" width="200" height="40" fill="none" stroke="black" stroke-width="1pt" stroke-dasharray="3pt" />
+    </g>
+  </defs>
+
+  <g transform="translate(300 300)">
+    <use href="#external" />
+    <text class="caption">Attachment</text>
+    <path class="relation from-many to-one" d="M 0,-20 0,-80" />
+  </g>
+    <g transform="translate(700 200)">
+    <use href="#model" />
+    <text class="caption">Recipient</text>
+    <path class="relation from-many to-one" d="M -100,0 -300,0" />
+    <text class="note" x="-120" y="20">1+</text>
+  </g>
+  <g id="message" transform="translate(300 200)">
```

Comment:
@martinmacko47 Zmenil som poradie modelov, pretoze v akom poradi su v subore, v takom poradi sa vykresluju. Ak by som zachoval poradie, relacia (rovna ciara) by mi prekreslila prazdne koliesko.
Je tu este moznost, ze by som skratil ciaru, alebo na konci suboru vykreslil model este raz pomocou `    <use xlink:href="#my-model"/>`

Reply:
Preco ciaru medzi `Message` a `Attachment` mas 2x? Samozrejme ze sa ti bude prekreslovat, ked ju mas tam dva krat cez seba. Je to jedna relacia, jedna ciara, tak ju treba vykreslit raz s markerom na oboch koncoch.

Ciary pre relacie by mali byt v `<g>` pri modeli v ktorom su v DB definovane. Pomocou class `from-*` a `to-*` znacime pocentnost relacie na oboch koncoch. Bezny FK je `from-many to-one`. FK pri ktorom je unique constraint je `from-optional to-one`. A podobne.

V tomto konkretnom pripade je relacia v DB definovana v `Attachment`, takze ju vykresli v `<g>` pri `Attachment` a pouzi classu `from-many to-optional`, kde `from-many` vykresli plnu gulicku na zaciatku ciary a `to-optional` prazdnu gulicku na konci ciary.

Reply:
:ok: